en/micro-integrator/docs/references/connectors/ISO8583-connector/ISO8583-inbound-endpoint-example.md

```diff
@@ -0,0 +1,106 @@
+# ISO8583 Inbound Endpoint Example
+
+The ISO8583 inbound endpoint of WSO2 EI acts as a message consumer. The ISO8583 inbound endpoint supported via the WSO2 EI is a listening inbound endpoint that can consume ISO8583 standard messages. 
+
+## What you'll build
+
+This sample demonstrates how the ISO8583 inbound endpoint working as ISO8583 message consumer. In this scenario to generate ISO8583 messages we used sample java client programme.
+
+The ISO8583 inbound endpoint listens on port 5000 and acts as a ISO8583 standard message consumer. When a sample java client connects on port 5000, the ISO8583 inbound endpoint consumes ISO8583 standard messages, converts the messages to XML format, and then injects messages to a sequence into the WSO2 EI.
+
+See [ISO8583 connector configuration](ISO8583-connector-configuration.md) for more information.However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 EI mediators. 
```

Comment:
```suggestion
See [ISO8583 connector configuration](ISO8583-connector-configuration.md) for more information. However, for simplicity of this example, we will just log the message. You can extend the sample as required using WSO2 EI mediators. 
```

Shall we add some link here on how to extend the sample using mediators?

Reply:
Added following link to the refer mediators [1].
[1] https://docs.wso2.com/display/EI660/ESB+Mediators